react/features/base/media/reducer.js

```diff
@@ -57,6 +59,43 @@ function _audio(state = AUDIO_INITIAL_MEDIA_STATE, action) {
     }
 }
 
+/**
+ * Initial state of the sounds reducer which is a {@code Map} of globally stored
+ * sounds.
+ *
+ * @type {Map<string, AudioElement>}
+ */
+const SOUNDS_INITIAL_STATE = new Map();
```

Comment:
I don't get that, can you be more specific ? Where would I put that ?

Reply:
I mean that the initial state can be just `undefined` because we do `new Map(oldState` and `new Map(undefined)` works as expected.

But maybe I missed something, since I reviewed this a while ago, so we can re-evaluate on the next round.

Reply:
I don't think this will work. This will be true only after one of the actions is reduced, but not before.